packages/web/src/pages/track-page/components/TrackPageLineup.tsx

```diff
@@ -88,7 +89,12 @@ export const TrackPageLineup = ({
   }
 
   const renderRemixesSection = () => {
-    if (indices.remixesSection.index === undefined || !trackId) return null
+    if (
```

Comment:
lets not drop this section until we are done with the submissions sub-section!

Reply:
was gonna gate this on feature flag, but realized we already flipped that one, might need another ff